src/release/breaking-changes/android-java-gradle-migration-guide.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Android Java Gradle migration guide
+description: How to migrate your Android app if you experience a run or build error from Gradle.
+---
+
+## Summary
+
+If you've recently upgraded Android Studio to the Flamingo
+release and have either run or built an existing Android app,
+you might have run into an error similar to the following:
+
+![Error dialog in Android Studio Flamingo]({{site.url}}/assets/images/docs/releaseguide/android-studio-flamingo-error.png){:width="80%"}
+
+The terminal output looks similar to the following:
+
+
+```sh
+FAILURE: Build failed with an exception.
+
+* Where:
+Build file '…/example/android/build.gradle'
+* What went wrong:
+Could not compile build file '…/example/android/build.gradle'.
+> startup failed:
+  General error during conversion: Unsupported class file major version 61
+
+  java.lang.IllegalArgumentException: Unsupported class file major version 61
+  	at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:189)
+  	at groovyjarjarasm.asm.ClassReader.<init>(ClassReader.java:170)
+  	[…
+  	 …
+  	 … 209 more lines of Groovy and Gradle stack trace …
+  	 …
+  	 …]
+  	at java.base/java.lang.Thread.run(Thread.java:833)
+```
+
+This error occurs because Android Studio Flamingo
+updates the Java SDK from 11 to 17, which
+isn't compatible with the version of the Java SDK
+used to build the app's Gradle file when the project
+was originally created with `flutter create`.
+
+**You can fix this error by upgrading your Gradle project
+to a compatible version (7.3 or later) using one of
+the following approaches.**
+
+## Solution #1: Guided fix using Android Studio
+
+Upgrade the Gradle version in Android Studio Flamingo
+as follows:
+
+**Step 1**: In Android Studio Flamingo, open the `android` folder,
+  which should bring up the following dialog: 
+
+![Dialog prompting you to upgrade Gradle]({{site.url}}/assets/images/docs/releaseguide/android-studio-flamingo-upgrade-alert.png){:width="50%"}
+
+**Step 2**: Follow the guided workflow to update Gradle:
+
+![Workflow to upgrade Gradle]({{site.url}}/assets/images/docs/releaseguide/android-studio-flamingo-gradle-upgrade.png){:width="85%"}
+
+## Solution #2: Manual fix at the command line
+
+Do the following from the top of your Flutter project.
+
+**Step 1**: Go to the gradle directory for your project:
+
+```sh
+cd android/gradle/wrapper
```

Comment:
````suggestion
```terminal
$ cd android/gradle/wrapper
````

Reply:
Yeah, we've generally used this for consistency across the site in indicating it's a terminal command, but I'm fine dropping its use here. 

And you're right we likely don't need a shell prompt at all here, since we don't suggest using a command to modify the properties file anyway.